Issuer/Issuer.cs

```diff
@@ -142,5 +179,91 @@ private static ClaimStatus ByteArray2ClaimStatus(byte[] value)
             if (value == null || value.Length == 0) return ClaimStatus.Nonexistent;
             return (ClaimStatus)(int)(BigInteger)(ByteString)value;
         }
+
+        /// <summary>
+        /// Add an admin
+        /// </summary>
+        /// <param name="pubKey"></param>
+        /// <returns>true if added successfully</returns>
+        public static bool AddAdmin(ECPoint pubKey)
+        {
+            var owner = GetOwner();
+            if (!Runtime.CheckWitness(owner)) throw new Exception("No authorization.");
+
+            if (AdminList[pubKey] is not null) throw new InvalidOperationException("The admin already exists.");
+            AdminList.Put(pubKey, 0);
+            return true;
+        }
+
+
+        /// <summary>
+        /// Remove an admin
+        /// </summary>
+        /// <param name="pubKey"></param>
+        /// <returns>true if removed successfully</returns>
+        public static bool RemoveAdmin(ECPoint pubKey)
+        {
+            var owner = GetOwner();
+            if (!Runtime.CheckWitness(owner)) throw new Exception("No authorization.");
+
+            if (AdminList[pubKey] is null) throw new InvalidOperationException("The admin does not exist.");
+            AdminList.Delete(pubKey);
+            return true;
+        }
+
+
+        /// <summary>
+        /// Get the current admin list
+        /// </summary>
+        /// <returns>The public keys of the admins</returns>
+        public static ECPoint[] GetAdminList()
+        {
+            // List<ECPoint> ret = null;
+            Iterator pubKeyList = AdminList.Find(FindOptions.RemovePrefix | FindOptions.KeysOnly);
+            int count = 0;
+            while (pubKeyList.Next())
+            {
+                count++;
+            }
+            ECPoint[] pubKeys = new ECPoint[count];
+            pubKeyList = AdminList.Find(FindOptions.RemovePrefix | FindOptions.KeysOnly);
+            int i = 0;
+            while (pubKeyList.Next())
+            {
+                pubKeys[i++] = (ECPoint)pubKeyList.Value;
+            }
+            return pubKeys;
+        }
+
+        /// <summary>
+        /// more efficient version of GetAdminList that uses a List so it only iterates AdminList once
+        /// </summary>
+        /// <returns>The public keys of the admins</returns>
+        public static List<ECPoint> GetAdminList2()
```

Comment:
I wasn't sure if it would be feasible to change the return value of GetAdminList from ECPoint[] to List<ECPoint> so I simply added a new method for now. If it's feasible to change, we should replace the logic of GetAdminList with the logic from GetAdminList2. Otherwise, we can remove GetAdminList2

Reply:
I also prefer to use GetAdminList2, since we have already supported the List type.

Reply:
I'll verify that it still works with the SDK and fold these into a single GetAdminList method the returns List<ECPoint> instead of ECPoint[]